### ISSUE TITLE
AttributeError: 'CosmosClient' object has no attribute 'ExecuteStoredProcedure'

### ISSUE THREAD

**Issue by sjhermanek** [Azure/azure-cosmos-python#196]
I'm trying to write a fairly simple piece of code that invokes a stored procedure in one of my containers.

```
# Required imports
from azure.cosmos import exceptions, CosmosClient, PartitionKey

# Initialize the Cosmos client
endpoint = "https://***.documents.azure.com:443/"
key = '***'

# Get a Cosmos client
client = CosmosClient(endpoint, key)

# Database name
database_name = 'SomeDBName'

# Collections to perform the hard delete
list_of_collections = ["SomeContainerName"]

# Name of sproc to invoke
sproc_name = "delete_by_query"

# Query to pass into the sproc
query = "select * from c where c.active = false"

# Invoke the sproc
for collection in list_of_collections:
    client.ExecuteStoredProcedure("dbs/{}/colls/{}/sprocs/{}".format(database_name, collection, sproc_name), query)
```

When running this, I get the following error:
```
AttributeError: 'CosmosClient' object has no attribute 'ExecuteStoredProcedure'
```

Please advise :)

**Comment by vchalmel** [comment]
Your import should be :
`from azure.cosmos.cosmos_client import CosmosClient`

**Comment by sjhermanek** [comment]
@vchalmel Just tried, and still running into the same error.

**Comment by vchalmel** [comment]
Which version of the package are you running ?

**Comment by sjhermanek** [comment]
I'm using version `4.0.0b6`

**Comment by southpolesteve** [comment]
@sjhermanek We don't have a sample, but here is a test showing how to execute a stored procedure in the v4 SDK https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/cosmos/azure-cosmos/test/test_crud.py#L622-L644

`execute_stored_procedure` is a method that exists on the container.scripts object. We just released 4.0 and are still working on getting all the docs updated.

If you run into any further issues, you can open an issue in our central Python repo https://github.com/Azure/azure-sdk-for-python. All active development has moved to that repo.
